io_test.go

```diff
@@ -288,50 +288,40 @@ func TestReadWriteWithCodec(t *testing.T) {
 	}
 }
 
-func TestWriteProtocolMessage(t *testing.T) {
-	req := &CancelRequest{
-		Request: Request{
-			ProtocolMessage: ProtocolMessage{
-				Type: "request",
-				Seq:  25,
-			},
-			Command: "cancel",
+const cancelMsgString = "Content-Length: 75\r\n\r\n{\"seq\":25,\"type\":\"request\",\"command\":\"cancel\",\"arguments\":{\"requestId\":24}}"
```

Comment:
nit: Msg => Req or Request (since you can also have response msg)

Reply:
Done